AUTHORS.txt

```diff
@@ -4,6 +4,8 @@ Francois BEAUDE
 Gautier BUREAU
 Angela CHIEH
 Sebastien DOUMEIX
+Lluc FIGUERAS
```

Comment:
Not related to this PR but should we also add Marco and I to this list ?

Reply:
yes !!

Reply:
Thank!
I think this has not been updated with your last commit.

Reply:
done!